app/components/point.jsx

```diff
@@ -65,6 +81,41 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
     alignSelf: 'stretch',
   },
 
+  // animation states
+  loadingAnimation: {
+    animationDuration: '1s',
+    animationFillMode: 'forwards',
+    animationIterationCount: 'infinite',
+    animationName: '$loadingAnimation_keyframes',
+    animationTimingFunction: 'linear',
+    background: '#f6f7f8',
+    backgroundImage: 'linear-gradient(to right, #eee 8%, #ddd 18%, #eee 33%)',
+    backgroundSize: 'inherit',
+    position: 'relative',
+    width: '100%',
+    marginBottom: '0.625rem',
+  },
+  loadingAnimationSubject: {
+    height: '2rem',
+  },
+  loadingAnimationDescription: { height: '1rem' },
+  '@keyframes loadingAnimation_keyframes': {
+    '0%': {
+      backgroundPosition: '-30rem 0',
```

Comment:
It scares me to see 30rem's here - but I see it's working at an arbitrary width.   Do you know, can you leave a comment about whey the 30rems and that it doesn't effect the size responsiveness of the point component.

Reply:
30rem is roughly equivalent to the amount of pixels in [the example](https://devsheet.com/post/skeleton-loading-elements-using-css/)

I'm wondering though, would it be okay to use a package instead? This one might be interesting:

https://www.npmjs.com/package/react-loading-skeleton